packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -188,6 +189,24 @@ export class PlaygroundWorkerEndpoint extends PHPWorker {
 		}
 
 		try {
+			if (shouldInstallWordPress === 'auto') {
```

Comment:
How could we make this reusable for Playground CLI?

Reply:
Both Playground remote and CLI leverage `bootWordPress()`, but AFAICT, everything before that is custom setup for their individual context. I'll consider whether there is more we can share, but at the very least, we could implement the same logic for CLI.

The CLI's `skipWordPressSetup` argument is a potential conflict and maybe could be deprecated in favor of a `shouldInstallWordPress` argument. Or they could be complementary.

Thanks for drawing my attention to this.

Reply:
> > How could we make this reusable for Playground CLI?
> 
> Both Playground remote and CLI leverage `bootWordPress()`, but AFAICT, everything before that is custom setup for their individual context. I'll consider whether there is more we can share, but at the very least, we could implement the same logic for CLI.

I was thinking of saying the following:
> To be able to implement this check in a way that works for both PHP and CLI, maybe such a feature should be moved into `bootWordPress()`. That way, we can write the check in PHP and...

But if we want to allow parallel downloads of PHP and WP resources in a web context, we need to check before PHP has been downloaded and loaded.

Reply:
I made a reusable `looksLikePlaygroundDirectory()` function under `@wp-playground/wordpress`. The relative nature of the `fileExists()` predicate feels a little clunky, but I think this is workable for sharing the looks-like-playground-dir heuristic.

```ts
/**
  * Check if the given directory handle directory is a Playground directory.
  *
  * @param fileExists Function A function that checks if a file exists relative to an assumed directory.
  * @returns Promise<boolean> Whether the directory looks like a Playground directory.
  */
 export async function looksLikePlaygroundDirectory(
 	fileExists: (relativePath: string) => Promise<boolean>
 ) {
 	const results = await Promise.all(
 		['wp-config.php', 'wp-content/database/.ht.sqlite'].map(fileExists)
 	);
 	return results.every(Boolean);
 }
```

Reply:
There's something similar in wp-now, you may want to review their "WordPress mode"

Reply:
Having support for this could improve the Playground tester's performance. 
Every test downloads WordPress and it's the majority of the test time. Being able to download WP once and mount it would be helpful.